README.md

```diff
@@ -158,6 +158,15 @@ This plugin creates device plugin endpoints based on the configurations given in
                 "devices": ["154c", "10ed"],
                 "drivers": ["vfio-pci"]
             }
+        },
+        {
+            "resourceName": "mlnx_sriov_rdma",
```

Comment:
it shouldn't say mlnx here.
vendor and device id describe what vendor it is.

resource name should be just 'rdmaDevice'.

Reply:
@paravmellanox this is an example for Mellanox's RDMA devices